src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -322,6 +331,10 @@ public enum ReactiveRangeCheckMode {
 
     private PhaseShifterControlMode phaseShifterControlMode = PHASE_SHIFTER_CONTROL_MODE_DEFAULT_VALUE;
 
+    private Set<ReportedFeatures> reportedFeatures = REPORTED_FEATURES_DEFAULT_VALUE;
+
+    private boolean detailedNrReport = false;
```

Comment:
This one is not needed anymore? A empty reportedFeatures means no additional reports?

Reply:
*A empty reportedFeatures means no additional reports?* Yes, it is verified via `.contains(key)`

*This one is not needed anymore?* In the way I coded it is still neded. The problem I faced is that inside the NR it doesn't know if it's running a LF or a SA. So I created this variable and I set it to

`parametersExt.getReportedFeatures().contains(OpenLoadFlowParameters.ReportedFeatures.NEWTON_RAPHSON_LOAD_FLOW)`

inside the creation of OLF parameters, and to `lfParametersExt.getReportedFeatures().contains(OpenLoadFlowParameters.ReportedFeatures.NEWTON_RAPHSON_SA))` 

inside the creation of SA parameters.

Reply:
It's really confusing to do this, you're hijacking the parameters class by doing this. No, you'd better find another way to do this, like passing the corresponding boolean in `OpenLoadFlowParameters::createAcParameters`, similarly to `forceA1Var` variable.

Reply:
@caioluke I pushed a design fix proposition. Are you ok with it?